prow/plugins/label/label.go

```diff
@@ -106,6 +106,9 @@ func getLabelsFromGenericMatches(matches [][]string, additionalLabels []string,
 		labelFilter.Insert(strings.ToLower(l))
 	}
 	for _, match := range matches {
+		for i := range match {
```

Comment:
Seems like we only ever use `match[0]` -- why do we need to apply this to all of them?